### PR TITLE
[clang] CWG150: add tests and change to unreleased

### PR DESCRIPTION
**Pull request by mizvekov**

### CODE REVIEW

File: clang/test/CXX/drs/cwg1xx.cpp [hunk 1]
```diff
@@ -753,6 +753,46 @@ namespace cwg148 { // cwg148: yes
 
 // cwg149: na
 
+namespace cwg150 { // cwg150: yes
```
Comment (Endilll) [inline clang/test/CXX/drs/cwg1xx.cpp:756]:
```suggestion
namespace cwg150 { // cwg150: 19
```

**Review by Endilll — CHANGES_REQUESTED** [review]
You should apply the suggested change, and run `make_cxx_dr_status` script. I don't expect it to introduce new changes, though, but we should be on a safe side here, and not rely on manual editing of `cxx_dr_status.html`.

**Review by Endilll — COMMENTED** [review]
This looks good to me now.
The contents of the test go over my head, though.

**Review by cor3ntin — APPROVED** [review]
LGTM, thanks!

**Pull request merged**